DesignTwitter.java

```diff
@@ -0,0 +1,96 @@
+/*
+Time Complexity:O(logN)
+Space Complexity: unsure
+*/
```

Comment:
Please refer video for clearer idea of complexity

Reply:
O(m) --- M is number of users. We are using two HashMaps and Priority Queue. Priority queue is of size 10 only so ignore. Among other Hashmaps we Are mapping followers and tweets to number of user. Is this better Avinav?

Reply:
Yes, I get it. thank you!!